Curtains/Curtains.Api/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Curtains.Application.Interfaces;
+using Curtains.Application.DTO;
+using Curtains.Domain.Models;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace Curtains.Api.Controllers
+{
+    /// <summary>
+    /// This controller <c> ReviewController </c> for CRUID operations in database
```

Comment:
Bad description.
As a developer who can use this controller i would like more information about this controller instead of CRUD